scripts/globals/bcnm.lua

```diff
@@ -967,13 +967,16 @@ function EventFinishBCNM(player, csid, option)
             local stat = player:getStatusEffect(dsp.effect.BATTLEFIELD)
             local bfid = stat:getPower()
             local item = getItemById(player, bfid)
+            local initiatorName = ""
+            _, initiatorName = player:getBattlefield():getInitiator()
```

Comment:
You don't have to initialize `initiatorName` before assigning it a value, this also leads to `_` being a global.

```lua
local initiatorId, initiatorName = player:getBattlefield():getInitiator()
```

Reply:
Okay, I'll do it that way.

Forgot to add Shami properly checking broken orbs anyway.

Reply:
Done.